pkg/apis/core/types.go

```diff
@@ -2187,9 +2187,9 @@ type SecretKeySelector struct {
 	Optional *bool
 }
 
-// EnvFromSource represents the source of a set of ConfigMaps
+// EnvFromSource represents the source of a set of ConfigMaps or Secrets
 type EnvFromSource struct {
-	// An optional identifier to prepend to each key in the ConfigMap.
```

Comment:
thanks for the doc change, please update the versioned type as well (in staging/src/k8s.io/api/core/v1/types.go) and regenerate API docs (`make update`)